public/js/views/management/add-pay-method.jsx

```diff
@@ -33,7 +35,7 @@ export default class AddPayMethod extends Component {
         <CardForm
           submissionErrors={this.props.cardSubmissionErrors}
           submitPrompt={gettext('Add')}
-          handleCardSubmit={(card) => this.handleCardSubmit(card)}
+          handleCardSubmit={this.handleCardSubmit.bind(this)}
```

Comment:
one thing I like about bind is that you don't have to update it when the signature changes. In this case, it actually took me a while to track down that the lambda wasn't passing the right args :(